src/pages/paintings.js

```diff
@@ -1,8 +1,94 @@
 import * as React from "react"
+import { Composition, Box } from "atomic-layout"
+import styled from "styled-components"
 import { DefaultLayout } from "../components/DefaultLayout"
 
+import Section from "../components/Section"
+import { Button } from "../components/Button"
+import one from "../images/one.png"
+import two from "../images/two.png"
+import three from "../images/three.png"
+import four from "../images/four.png"
+
+const StyledBox = styled.div`
+  text-transform: uppercase;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  align-items: center;
+  margin-top: var(--spacing-20);
+`
+
+const StyledTitle = styled.h3`
```

Comment:
`h3` elements will look the same on the website. Don't write their styles here, instead write them in your `style.css`.